replicator/active_replicator_config.go

```diff
@@ -15,13 +17,19 @@ const (
 // ActiveReplicatorConfig controls the behaviour of the active replicator.
 // TODO: This might be replaced with ReplicatorConfig in the future.
 type ActiveReplicatorConfig struct {
-	ID string
-	// Type of replication: PushAndPull, Push, or Pull
+	ID     string
+	Filter string
+	// Since represents the sequence we're going to perform the replication from.
+	Since uint64
+	// ChangesBatchSize controls how many revisions may be batched per changes message.
+	ChangesBatchSize uint16
+	// Direction, otherwise known as the type of replication: PushAndPull, Push, or Pull.
 	Direction ActiveReplicatorDirection
-	// // CheckpointInterval controls many revisions to process before storing a checkpoint
-	// CheckpointInterval uint16 // Default: 200
-	// // ChangesBatchSize controls how many revisions may be batched per changes message
-	// ChangesBatchSize uint16 // Default: 200
-	// TargetDB represents the full Sync Gateway URL, including database path, and basic auth credentials
-	TargetDB *url.URL
+	// Continuous specifies whether the replication should be continuous or one-shot.
+	Continuous bool
+	// PassiveDB represents the full Sync Gateway URL, including database path, and basic auth credentials of the target.
+	PassiveDB *url.URL
```

Comment:
It would probably be clearer if this was named PassiveDBURL - otherwise it's unexpected that PassiveDB and ActiveDB aren't the same type.